web-ui/src/main/resources/catalog/views/default/templates/recordView.html

```diff
@@ -536,6 +536,12 @@ <h2 data-translate="">metadataInformation</h2>
           <a data-ng-href="https://www.facebook.com/sharer.php?u={{mdView.current.record.encodedUrl}}"
              target="_blank"
              class="btn btn-lg btn-default"><i class="fa fa-fw fa-facebook"></i></a>
+          <a data-ng-href="http://www.linkedin.com/shareArticle?mini=true&amp;url={{socialMediaLink}}" 
```

Comment:
Would have been better to drop encodedUrl and use socialMediaLink everywhere. Will do the change.